modules/cnv-deleting-vms.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-delete-vms.adoc
+
+[id="cnv-deleting-vms_{context}"]
+
+= Deleting a virtual machine and PVCs using the CLI
+
+When you delete a virtual machine, the PVC it uses is unbound. If you
+do not plan to bind this PVC to a different VM, delete it, too.
+
+You can only delete objects in the project you are currently working in,
```

Comment:
~only~ delete objects in only the project
I'd flip the clause order.

Reply:
I'd probably add this information to the steps themselves, too.

Reply:
Did flip; moved that information into the procedure as an all-encompassing NOTE.